src/components/board/board.tsx

```diff
@@ -45,6 +51,12 @@ export const Board: FC = () => {
 
   useEffect(() => {
     updateColumns();
+
+    return () => {
+      const resetFunc = (): void => setColumns(null);
+
+      dispatch(boardActions.reset(resetFunc));
+    };
```

Comment:
Can we do this in this way?
```suggestion
    updateColumns();

    return () => {
      dispatch(boardActions.reset());
    };
```

and then in useEffect
```
useEffect(() => {
  setColumns(board?.columns ?? []);
}, [board?.columns]);
```

Reply:
I dont need another updating useEffect because i have updateColumns